core/java/iesi-core/src/main/java/io/metadew/iesi/common/crypto/FrameworkCrypto.java

```diff
@@ -19,15 +22,28 @@
 
     private static FrameworkCrypto INSTANCE;
 
-    public static FrameworkCrypto getInstance() {
+    public static FrameworkCrypto getInstance() throws Exception {
         if (INSTANCE == null) {
             INSTANCE = new FrameworkCrypto();
         }
         return INSTANCE;
     }
 
-    private FrameworkCrypto() {
-        this.aes = new AESEncryptBasic("c7c1e47391154a6a");
+    private FrameworkCrypto() throws Exception {
+        Configuration configuration = Configuration.getInstance();
+        if (configuration.getProperty("iesi.security.encryption.key").get() != null) {
+            this.aes = new AESEncryptBasic(configuration.getProperty("iesi.security.encryption.key").get().toString());
+        } else if (configuration.getProperty("iesi.security.encryption.alias").get() != null) {
+            BufferedReader br = new BufferedReader(new InputStreamReader(System.in));
+            System.out.println("Enter your password: ");
+            String password = br.readLine();
+            String keystoreLocation = configuration.getProperty("iesi.security.encryption.keystore-path").get().toString();
```

Comment:
Could you do it as follows: https://stackoverflow.com/questions/8138411/masking-password-input-from-the-console-java